### PR TITLE
Update plugin to use respec-vc

### LINKED ISSUE #243: Move example rendering to respec-vc
The code housed [here](https://github.com/w3c/vc-jose-cose/tree/main/plugin) pertaining to our plugin for rendering vc-jose-cose examples should be moved to [respec-vc](https://github.com/w3c/respec-vc) for use here and in the VC Data Model itself.

### PR DESCRIPTION
**Pull request by decentralgabe**
Fix #243 

This will remain in draft until the bug relating to media types for rendering JWTs is fixed.

Also should wait until #279 is merged.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jose-cose/pull/280.html" title="Last updated on Jul 5, 2024, 4:27 PM UTC (29c6be2)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jose-cose/280/46c4e0e...29c6be2.html" title="Last updated on Jul 5, 2024, 4:27 PM UTC (29c6be2)">Diff</a>

### CODE REVIEW

**Comment by decentralgabe** [comment]
Blocked by https://github.com/w3c/respec-vc/pull/32

**Comment by decentralgabe** [comment]
This is nearly ready, but there is a bug that I have opened https://github.com/w3c/respec-vc/pull/36 to address. Once this goes in this can be merged.

**Comment by decentralgabe** [comment]
updated and ready for merge

**Review by selfissued — APPROVED** [review]

**Pull request merged**
